### PR TITLE
Update ring.markdown

### PR DESCRIPTION
**Pull request by Robbot**
## Proposed change

This change in documentation should let no coding users understand better, the downloaded files location, issues with configuration.   

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Robbot, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21724).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/ring.markdown [hunk 1]
```diff
@@ -80,6 +80,19 @@ automation:
 
 {% endraw %}
 
+You may consider some modifications in the subdir and the filename to suit your needs. For example you can add the date and the time and extension to the downloaded file:
```
Comment (frenck) [inline source/_integrations/ring.markdown]:
```suggestion
You may consider some modifications in the subdirectory and the filename to suit your needs. For example, you can add the date and the time and extension to the downloaded file: 
```

File: source/_integrations/ring.markdown [hunk 2]
```diff
@@ -80,6 +80,19 @@ automation:
 
 {% endraw %}
 
+You may consider some modifications in the subdir and the filename to suit your needs. For example you can add the date and the time and extension to the downloaded file: 
+
+{% raw %}
+```yaml
+    data:
+      url: "{{ state_attr('camera.front_door', 'video_url') }}"
+      subdir: "{{state_attr('camera.front_door', 'friendly_name')}}/{{ now().strftime("%m.%Y") }}"
```
Comment (frenck) [inline source/_integrations/ring.markdown]:
```suggestion
      subdir: "{{ state_attr('camera.front_door', 'friendly_name') }}/{{ now().strftime("%m.%Y") }}"
```

File: source/_integrations/ring.markdown [hunk 3]
```diff
@@ -80,6 +80,19 @@ automation:
...
+      filename: "{{now().strftime("%Y-%m-%d-at-%H-%M-%S") }}.mp4}"
```
Comment (frenck) [inline source/_integrations/ring.markdown]:
```suggestion
      filename: "{{ now().strftime("%Y-%m-%d-at-%H-%M-%S") }}.mp4"
```

File: source/_integrations/ring.markdown [hunk 4]
```diff
@@ -80,6 +80,19 @@ automation:
...
+```
+{% endraw %}
+
+the above modification will save the video file under `<config>/downloads/<camera_name>/YYYY-MM/YYYY-MM-DD-at-HH-MM-SS.mp4`. You can change the date according to your language format.
```
Comment (frenck) [inline source/_integrations/ring.markdown]:
```suggestion
the above modification will save the video file under `<config>/downloads/<camera_name>/YYYY-MM/YYYY-MM-DD-at-HH-MM-SS.mp4`. You can change the date according to your localization format.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @Robbot 👍 

Left some comments, could you take a look? 🙏 

../Frenck

**Review by frenck — APPROVED** [review]
Thanks, @Robbot 👍 
Nice first-time contribution 🥇 

../Frenck

**Pull request merged**